docs/e2e_tests.md

```diff
@@ -0,0 +1,40 @@
+---
+title: E2E tests of btp-manager installation from OCI module image
+---
+
+## Overview
+
+End to end (E2E) tests currently check if we are able to install and then uninstall btp-manager using OCI module image and helm.
+The flow is as follows:
```

Comment:
Wouldn't it be better to have a numbered list here?

Reply:
Changed in the next commit.